Add limitation to embedded documentation (#381)
vaadin/docs: articles/flow/integrations/embedding/limitations.asciidoc

----
+
*** This example assumes that the embedding (host) site is served from the same host mapped to port 80 (be it a servlet container or a custom Python HTTP server). Our servlet container with our Vaadin servlet is bound to, for example, 8080.

> * Theme resources should be available on the server of the embedding page or inlined in css.
> +
> When embedding the application on a different server any relative css url will target the embedding server and not the embedded application server.

Reviewer: ```suggestion
* Theme resources should be available on the server of the embedding page or embedded in CSS (as data URLs).
+
When embedding the application on a different server, relative CSS URLs target the embedding server and not the embedded application server.
```

----
+
*** This example assumes that the embedding (host) site is served from the same host mapped to port 80 (be it a servlet container or a custom Python HTTP server). Our servlet container with our Vaadin servlet is bound to, for example, 8080.
* Theme resources should be available on the server of the embedding page or embedded in CSS (as data URLs).
+
When embedding the application on a different server, relative CSS URLs target the embedding server and not the embedded application server.
+

> In java code the target server can be gotten from the request as `StringBuffer requestURL = ((HttpServletRequest) VaadinRequest.getCurrent()).getRequestURL();` which should then be added before the resource.

Reviewer: ```suggestion
Retrieve the target server address from the request as follows:

[source,java]
----
StringBuffer requestURL = ((HttpServletRequest) VaadinRequest.getCurrent()).getRequestURL();
----

Then you can prepend that to the resource URL. For example:
```

> For instance a logo would be gotten as `new Image(requestURL.toString() + "themes/app-theme/img/logo.png", "logo");`.

Reviewer: ```suggestion
[source,java]
----
new Image(requestURL.toString() + "themes/app-theme/img/logo.png", "logo");
----
```
